CitMin/Languages/en.ini

```diff
@@ -100,22 +100,22 @@ FastAndSlow=Fast And Slow
 WAIEIDifficulty=Select difficulty.
 WAIEIOptions=WAIEI's own options.
 
-UnJoin=The player who presses the decision button will unjoin. This feature is only available in two-player mode.
+UnJoin=The player who presses the start button will unjoin. This feature is only available in two-player mode.
 ContinueCombo=Set the lowest judgments to continue the combo.
 ComboColor=Set the coloring rules for the combo numbers. If StepMania is selected, once the combo is cut, it is fixed to white.
-BreakComboMineHit=Set whether to combo break when Mine is hit.
-BreakComboHoldLetGo=Set whether to combo break when Hold or Roll is N.G.
-RollCombo=Set whether or not to add combos for the number of times you hit Roll in a row.
-JudgmentsUnderField=JudgmentsSets the Judgment's drawing priority to below the Notes.
-ComboUnderField=Set the combo's drawing priority to below the note.
+BreakComboMineHit=Set whether to combo break when Mines/Shock Arrows are hit.
```

Comment:
This is only a theme for StepMania/Project OutFox, so please do not use the original DDR term "shock arrow".
In general, only the terms used in the Lua documentation can be used.
However, as a special exception, we use the notation "W1 (Marvelous)" to "W5 (Almost)", because "W1" to "W5" are not understood by users.

Reply:
Shoot, I just saw this now, I'll fix that. Thank you